src/extension/geofence-server/src/main/java/org/geoserver/geofence/server/rest/RulesRestController.java

```diff
@@ -206,7 +208,9 @@ public JaxbRuleList count(
             adminService.setDetails(id, rule.getLayerDetails().toLayerDetails(null));
         }
 
-        return new ResponseEntity<Long>(id, HttpStatus.CREATED);
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.TEXT_PLAIN);
```

Comment:
These headers aren't used.

Reply:
Right, the change in "produces" does the trick, not this bit, I'll remove.